### DISCUSSION TITLE
Gold Egg boost?

### DISCUSSION THREAD

**Discussion by IvanSkazkin** [sunflower-land/sunflower-land#2057] [Q&A]
Can you tell me something I don't understand?
The golden egg has very poor boost.
We waited a year and they gave us this. I understand the t3 beaver and the t3 scarecrow. Their boost was known in advance.
The egg has been waiting for a year, and the boost is worse than that of t3 things.

With the green amulet, I don't need that golden rooster at all.
I make cakes every day and I have enough wheat, even with a large supply.
This is a super bad buff.

If there was no need for wheat at all, that would be a different matter.

In the future there will be more beds and therefore even if there are more animals, I don't think I can achieve a shortage of wheat.

**Comment by brian-man** [comment]
> The golden egg has very poor boost.

Disagree.

It's a direct peer for T3:
- Kuebiko => free crop production
- Foreman Beaver => free wood production
- Gold Egg => free egg production

**Comment by Dragonbask** [comment]
This is a great buff that will grow as you obtain more chickens it makes eggs essentially free to you. Comparing it to  an amulets ability to farm wheat that you still pay for is very narrow/closed minded. The buff is great and a bit op as a 300 supply nft should be.

**Reply by IvanSkazkin** [reply]
> Это отличный бафф, который будет расти по мере того, как вы будете получать больше цыплят, он делает яйца практически бесплатными для вас. Сравнивать это со способностью амулетов выращивать пшеницу, за которую вы все еще платите, очень узкое/закрытое мышление. Бафф отличный и немного активный, как и должен быть 300 nft.

Okay, let's assume the boost is good.
Why release it now when there is no use for it at all. We only have a few chickens.
Why not hatch it when the cows or bees come out.

If he added another 25%+ discount - it would be a good boost for cakes

**Reply by Dragonbask** [reply]
I don't get where you are saying it is not useful now? eggs make cakes that give exp. Most everything players will want to own or work towards is going to be locked behind a level requirement. also at level 21 we can now add another hen house for more chickens. Ask any top players they leveled and will continue to level with cake as one of there main foods to cook and eat.

**Reply by IvanSkazkin** [reply]
> I don't get where you are saying it is not useful now? eggs make cakes that give exp. Most everything players will want to own or work towards is going to be locked behind a level requirement. also at level 21 we can now add another hen house for more chickens. Ask any top players they leveled and will continue to level with cake as one of there main foods to cook and eat.

I'm already lvl 30 and I have a bunch of cake. 550 borscht, 30+ cakes and much more. There are many more resources. It's all because of the green amulet. The green amulet makes the boost useless. He also needs only chickens. Not all animals

**Reply by Dragonbask** [reply]
The boost makes egg production free. you'll never pay to feed chickens again. I don't get how the amulet makes this worth less lol
